index.html

```diff
@@ -334,28 +330,70 @@ <h2>Get Started</h2>
+          <img class="animate" src="./images/linkedin.png" alt="animating-image" />
 
-          <img class="animate" src="./images/Vector3.png" alt="" />
-          <img class="animate" src="./images/Vector4.png" alt="" />
+          <img class="animate" src="./images/Vector3.png" alt="animating-image" />
+          <img class="animate" src="./images/Vector4.png" alt="animating-image" />
         </div>
       </div>
     </footer>
+            <!--modal window-->
+            <div class="modal" id="my-modal">
+              <div class="modal-content" >
+                <div class="modal-headline">
+                  <span class="close">&times;</span>
+                  <h2>project  name goes here</h2></div>
+                <div class="modal-list">
+                  <ul>
+                    <li><a href="#">HTML/CSS</a></li>
+                    <li><a href="#">Ruby on Rails</a></li>
+                    <li><a href="#">Javascript</a></li>
+                  </ul>
+                </div>
+                <div class="modal-img">
+                  <div class="" >
+                    <img id="desktop1" src="./images/desktop2.png" alt="modal-image"></div>
+                  <div class="small-img">
+                    <img id="desk1" src="./images/desktop2.png" alt="modal-image">
+                    <img id="desk2" src="./images/desktop2.png" alt="modal-image">
+                    <img id="desk3" src="./images/desktop2.png" alt="modal-image">
+                    <img id="desk4" src="./images/desktop2.png" alt="modal-image">
+                    <img id="desk5" src="./images/desktop2.png" alt="modal-image">
+                  </div>
+                  <div>
+                  </div>
+                </div>
+                <div class="modal-para">
+                  <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eveniet quam provident explicabo velit. Laboriosam ipsa veritatis ipsam asperiores nulla distinctio doloribus atque nobis sed recusandae. Necessitatibus ipsam veniam dolorem sapiente!</p>
+                  <p> Minima odio nisi animi at praesentium quos odit possimus reprehenderit repellat, ipsam saepe est assumenda fugiat qui deleniti sed similique nam tenetur harum quisquam ducimus. Similique eum ab deserunt voluptatibus?</p>
+                </div>
+                <div class="modal-btn">
+                  <button type="button" id="modal-btn1">see live <img id="cross-img4" src="./images/Union8.png" alt=""></button>
+                  <button type="button" id="modal-btn2">see source<img id ="cross-img3" src="./images/Vector1.png" alt=""></button>
+                </div>
+                <div class="modal-link">
+                  <a href=""> &#8592;  <span>previous project</span></a>
+                  <a href="">next project <span>&#8594;</span> </a>
+                </div>
+              </div>
+            </div>
+    <!--modal pop up ends here-->
```

Comment:
Your model should also be generated dynamically using Javascript. Kindly update this section also.
Also, your model is spanning more than two pages vertically, and margins, padding, spacing, and positioning need to be almost exactly like the Figma template of your choosing. Kindly make it according to your Figma design.